src/StakingNode.sol

```diff
@@ -399,56 +463,90 @@ contract StakingNode is IStakingNode, StakingNodeEvents, ReentrancyGuardUpgradea
         // 2. For beaconChainETHStrategy, the DelegationManager calls _withdrawSharesAsTokens interacts with the EigenPodManager.withdrawSharesAsTokens
         // 3. Finally, the EigenPodManager calls withdrawRestakedBeaconChainETH on the EigenPod of this StakingNode to finalize the withdrawal.
         // 4. the EigenPod decrements withdrawableRestakedExecutionLayerGwei and send the ETH to address(this)
-        delegationManager.completeQueuedWithdrawals(withdrawals, tokens, middlewareTimesIndexes, receiveAsTokens);
+        delegationManager.completeQueuedWithdrawals(withdrawals, tokens, receiveAsTokens);
 
         uint256 finalETHBalance = address(this).balance;
         uint256 actualWithdrawalAmount = finalETHBalance - initialETHBalance;
 
-        // NOTE: actualWithdrawalAmount may be < totalWithdrawalAmount in case of slashing !
-
-        // Shares are no longer queued; decrease what was queued for withdrawal
-        queuedSharesAmount -= totalWithdrawalAmount;
+        if (actualWithdrawalAmount != totalWithdrawableShares) {
```

Comment:
the deleted comment and cod is still valid to my knowledge

```actualWithdrawalAmount may be < totalWithdrawalAmount in case of slashing```

what happens here when there is beacon chain slashing?

How was this tested?